minitwit.py

```diff
@@ -98,6 +98,7 @@
 #Read config settings from the file specified by this env var, if it is defined.
 app.config.from_envvar('MINITWIT_SETTINGS', silent=True)
 
+secrets_used = []
```

Comment:
I can't set a global, but I can update the contents of a global list. There's probably a nicer way, like via the namedtuple that returns the creds... thoughts?